include/sdf/ParserConfig.hh

```diff
@@ -215,6 +215,10 @@ class SDFORMAT_VISIBLE ParserConfig
   /// and merge the child link into the parent.
   public: bool URDFPreserveFixedJoint() const;
 
+  /// \brief Set the storeResolvedURIs flag value.
+  /// \sa SetStoreResolvedURIs
+  public: void SetStoreResovledURIs(bool _resolveURI);
```

Comment:
We can deprecate this now.

Reply:
deprecated with `GZ_DEPRECATED(15)` and force pushed